Foundation/Block Enumeration/TWTBlockEnumeration.h

```diff
@@ -72,18 +80,31 @@ typedef id (^TWTBlockEnumerationInjectBlock)(id memo, id element);
  @result A new instance of the collection with the results of invoking the block on each element of
     the original collection.
  */
-- (instancetype)twt_collectWithBlock:(TWTBlockEnumerationCollectBlock)block;
+- (id)twt_collectWithBlock:(TWTBlockEnumerationCollectBlock)block;
```

Comment:
These changes are because the methods don’t return `instancetype` for `NSEnumerator`.